### ISSUE TITLE
NVDA and Google Voice don't work

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1305]
**Reported by KevanGC on 2011-01-07 08:18**
NVDA doesn't work well with Google Voice.

When I go to select a number or change settings, NVDA goes into this weird dialog thingy and I keep having to press CTRL+Insert+Space to get back to the regular screen. I'm using Firefox 3.6.13.

**Comment by ehollig** [comment]
Google Voice has gone through a few changes over the years. Considering the age of this ticket, lack of information and steps to reproduce, I think it is safe to say it is fixed.

**Comment by leonardder** [comment]
Hmm, saying that it has been fixed goes a bit too far in my opinion since we aren't sure. Closing, but I think neither works for me nor invalid is appropriate here.
